dwl.c

```diff
@@ -442,6 +444,15 @@ cleanupmon(struct wl_listener *listener, void *data)
 	free(m);
 }
 
+void
+commitnotify(struct wl_listener *listener, void *data)
+{
+	Client *c = wl_container_of(listener, c, commit);
+
+	/* client is dirty if there are any changes that have not been committed */
+	c->dirty = c->xdg_surface->configure_serial != c->xdg_surface->configure_next_serial;
```

Comment:
This would pause rendering for any configure-able property, and not just resizes.  Don't set the flag here, only clear it:
```suggestion
	c->dirty = c->dirty && c->xdg_surface->configure_serial != c->xdg_surface->configure_next_serial;
```
or
```suggestion
	if (c->xdg_surface->configure_serial != c->xdg_surface->configure_next_serial)
		c->dirty = 0;
```
(`&=` would work but is a little too tricky for my taste, and `&&=`, sadly, doesn't exist)

Reply:
Specifically targets the configure serial of the resize now.